src/applications/personalization/account/components/AccountVerification.jsx

```diff
@@ -15,8 +15,8 @@ export default function AccountVerification({ loa }) {
               status or send secure messages to your health care team?
             </h4>
             <p>
-              We need to make sure you’re you-and not someone pretending to be
-              you-before we can give you access to your personal and
+              We need to make sure you’re you—and not someone pretending to be
```

Comment:
These don't look like em dashes to me :) `—`

Reply:
It's a dumb text-editor display thing (incl. browser dev-tools too); browser webpage displays em-dashes properly:
![17769-em-dash-display](https://user-images.githubusercontent.com/587583/56384194-b0464d80-61d0-11e9-8739-096298f37a7e.png)